### ISSUE TITLE
How to run the services as standalone in window 10

### ISSUE THREAD

**Issue by shyamnayak6** [swiftops/jmeter-performance-report-parser#1]
Hi,

Can you explain if it is possible to run the services in window system.

Thanks,
Shyam

**Comment by deenaik** [comment]
Yes, you can since it is written in python. you just need a python 3.6 or above installed in your system.

**Comment by deenaik** [comment]
Yes, you can since it is written in python. you just need a python 3.6 or above installed in your system.

**Comment by avarne** [comment]
Yes, you can. You need to have Python 3.6 or above installed on your windows machine. Also you need mongo db setup for storing the result for perf run. Read Installation part for executing service standalone.
